sql/data/anime/madman/dragonball_z_data.sql

```diff
@@ -1,51 +1,43 @@
 -- Anime Data
 INSERT INTO ANIME(ANIME_TITLE, NUMBER_OF_EPISODES, ANIME_SYNOPSIS, ANIME_DESCRIPTION, COMPANY_NAME, COPYRIGHT) VALUES
-		("Title",
-		 n,
-		 "Synopsys",
-		 "Description",
-		 "Company",
-		 "Copyright"
+		("Dragon Ball Z Remastered Uncut Season 1",
+		 291,
```

Comment:
Dragonball Z has 39 episodes in first season, each season should be done indervidually (as with other sequal series)